.gitignore

```diff
@@ -1,3 +1,3 @@
 composer.lock
 vendor
-.idea
\ No newline at end of file
+.idea
```

Comment:
This belongs in a global .gitignore

https://murze.be/2014/12/create-a-global-gitignore/